carp.common/src/commonMain/kotlin/dk/cachet/carp/common/data/CarpDataTypes.kt

```diff
@@ -46,5 +46,23 @@ object CarpDataTypes : EnumObjectList<DataType>()
      * Heart rate. Measures the number of heart contractions (beats) per minute.
      */
     val HEARTRATE = add( DataType.fromFullyQualifiedName( HEARTRATE_TYPE_NAME ) )
+
+    internal const val RRI_TYPE_NAME = "$CARP_NAMESPACE.rri"
+    /**
+     * Heart rate as RRI. Measures the R to R wave interval in ms.
```

Comment:
Don't forgot to update documentation accordingly. 😉

Reply:
fixed